ui/src/main/java/com/xmartlabs/bigbang/ui/BaseActivity.kt

```diff
@@ -2,22 +2,16 @@ package com.xmartlabs.bigbang.ui
 
 import android.content.Context
 import android.os.Bundle
-import bullet.ObjectGraph
 import com.f2prateek.dart.Dart
 import com.trello.rxlifecycle2.components.RxActivity
-import com.xmartlabs.bigbang.core.Injector
 
-/**
- * A base Activity that inherits from [RxActivity] and performs [Dart] and [ObjectGraph] injections
- * in the [.onCreate] lifecycle method.
- */
+/** A base Activity that inherits from [RxActivity] and performs [Dart] in the [.onCreate] lifecycle method. */
```

Comment:
`and performs [Dart] in the` should be `and performs a [Dart] injection in the` (was missing the word injection)